apps/admin-portal/src/pages/applications.tsx

```diff
@@ -52,14 +54,15 @@ const APPLICATIONS_LIST_SORTING_OPTIONS: DropdownItemProps[] = [
     }
 ];
 
-const DEFAULT_APP_LIST_ITEM_LIMIT: number = 10;
+// TODO: Calculate based on the screen dimensions.
+const DEFAULT_APP_LIST_ITEM_LIMIT = 10;
 
 /**
  * Overview page.
  *
- * @return {JSX.Element}
+ * @return {React.ReactElement}
```

Comment:
Remove `React` since the component is already imported and used.

Reply:
Fixed with commit `e941ae3`